app/views/layouts/application.html.erb

```diff
@@ -4,6 +4,14 @@
     <title>WNB.rb</title>
     <meta name="viewport" content="width=device-width,initial-scale=1">
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Rubik:200,300,400,500,600,700,800,900">
+    <script async src="https://www.googletagmanager.com/gtag/js?id=<%= ENV['GA_ID'] %>"></script>
```

Comment:
The Google Analytics ID isn't a secret (and anybody would be able to see ours by inspecting the source code on the site) but I figure it's better to keep this stuff in an environment variable in case we have to switch it/create a staging version of the site/etc.

Reply:
That's interesting. I wonder if folks ever attack other sites' google analytics pages.

Reply:
My guess is Google prevents this by asking you to specify the domain in your account. I've put `https://wnb-rb.dev` as the domain, so requests that come from any other site wouldn't be counted.